docs/getting-started/concepts.md

```diff
@@ -7,67 +7,67 @@ Iter8 is the Kubernetes release optimizer built for DevOps, MLOps, SRE and data
 
 ## Use-cases
 
-Iter8 simplifies a variety of traffic engineering and metrics-driven validation use-cases. They are illustrated below.
+Iter8 simplifies a variety of traffic engineering and metrics-driven validation use-cases. To support such use cases, Iter8 provides support for the key challenges enabling simpler implementation and quicker adoption.
 
-![Iter8 use-cases](images/iter8usecases.png)
+**Progressive release with automated traffic management** Iter8 supports blue-green and canary release of new application and ML models. When new models are deployed, Iter8 automatically reconfigures the routing to desired traffic pattern. Deployment of new versions and their promotion is done by describing the desired state.
 
-## Design
+**A/B/n testing with client SDK and business metrics** Iter8 addresses the challenge of doing A/B/n testing of backend application components/ML models. It provides a simple client SDK allowing a user-facing component to easily reliably associate business metrics with the backend components that are used. This SDK provides stick lookup based on user request headers.
 
-Iter8 provides three inter-related components to support the above use-cases.
+**Performance testing for HTTP and gRPC endpoints** To enable rapid testing, Iter8 provides synthetic load generation and mechanisms to support test notifications. A set of reusable tasks can be used to implement the desired test and notification behavior.
 
-1. Iter8 controller
-2. Client SDK
-3. Composable tasks
+## Design Principles
 
-=== "Iter8 controller"
+**Support all application** Iter8 does not limit what types of resources define an application. It supports applications that are composed of any Kubernetes resources including those defined by custom resource definitions (CRDs). Adding support for a new resource type is both straightforward and declarative.
```

Comment:
```suggestion
**Support all application** Iter8 does not limit what types of resources an application is composed of. It supports applications that are composed of any Kubernetes resources including those defined by custom resource definitions (CRDs). Adding support for a new resource type is both straightforward and declarative.
```

Even though using define "allows: for a shorter sentence, we use "define" in plenty more places. Wondering if this word change can decrease the likelyhood of misinterpreting the sentence.